Workshop_1/Leksjon_3/README.md

```diff
@@ -92,7 +93,7 @@ I solution ligger en klasse som heter StorageService hvor det meste av funksjona
   Sett så opp BlobSasBuilder med riktige permissions. Det som må settes er:
```

Comment:
har begynt å sett over leksjon 2, og sett litt over leksjon 1 igjen, det ser ut som at blob container sin public access allerede blir satt opp med Blob (anonymous read access for blobs only). Linje 47. 

Så tror ikke vi behøver å generere sas.

Reply:
Men en annen måte er at vi kanskje heller setter opp container med private access, slik at man faktisk må generere sas token ?

Reply:
Egentlig enig i det siste, tror det er et poeng med SAS-tokens siden vi gir URL direkte til frontend som henter bilder selv.  Kan endre access

Reply:
Fikset tekst, Mari fikser js-kode